packages/generator/src/vdm-types.ts

```diff
@@ -1,8 +1,10 @@
 /* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
 
 import { PathLike } from 'fs';
+import { ODataVersion } from '@sap-cloud-sdk/util';
 
 export interface VdmServiceMetadata {
+  oDataVersion: ODataVersion;
```

Comment:
Isn't this addition and the `isCollection` a breaking change?  I assume nobody will use this object outside of our code but if you see some risk we should do the usual depreciation and add a new object.

Reply:
We could also make this optional and add default values, wherever this is passed. I don't love that and would rather prefer to add a compatibility note. What do you think?

Reply:
as discussed we go for a note since it is not really a interface for the outside world and the usage should be limited and we take the risk over making the properties optionals with defaults since this is not a good concept to do just for avoid a breaking change.